common/src/main/java/dev/aurelium/auraskills/common/source/type/BlockSource.java

```diff
@@ -24,9 +25,10 @@ public class BlockSource extends Source implements BlockXpSource {
     private final boolean trunk;
     private final boolean leaf;
```

Comment:
If we are no longer using the trunk and leaf options in `Treecapitator`, can you remove these fields, the constructor params, the getters, and the `BlockSourceParser` parsing for them?